docs/crud/crudruntime.md

```diff
@@ -1,78 +1,39 @@
 ---
 id: crudruntime
-title: CRUD Runtime abstraction
+title: CRUD Runtime Abstraction
 sidebar_label: CRUD Abstraction
 ---
 
 ## Graphback CRUD Abstraction
 
-Graphback CRUD abstraction available as part of the @graphback/runtime package provides a way to programatically define different datasources and configure them without code generation.
+A Graphback CRUD abstraction is available as part of the `@graphback/runtime` package, providing a way to programatically define different datasources and configure them without code generation.
 
 Runtime is abstracted into two interfaces:
 
-- `GraphbackCRUDService` - layer that wraps data provider, can implement custom behaviour like subscriptions or live queries
-- `GraphbackDataProvider` - data access layer 
+- `GraphbackCRUDService`: layer that wraps data provider, can implement custom behaviour like subscriptions or live queries
+- `GraphbackDataProvider`: data access layer 
 
 ### GraphbackCRUDService
 
-Graphback provides following implementations of GraphbackCRUDService
+Graphback provides the following implementations of `GraphbackCRUDService`
 
-- `CRUDService` - implements subscriptions and logging. Used by default in resolver generator.
+- `CRUDService`: implements subscriptions and logging. Used by default in resolver generator
 
 ### GraphbackDataProvider
 
-Graphback provides following implementations of GraphbackDataProvider
+Graphback provides the following implementations of `GraphbackDataProvider`:
 
-- PgKnexDBDataProvider (PostgreSQL version from `@graphback/runtime-knex`)
-- MongoDBDataProvider (`@graphback/runtime-mongodb`)
+- [`PgKnexDBDataProvider`](https://www.npmjs.com/package/@graphback/runtime-knex): PostgreSQL data provider using [Knex.js](http://knexjs.org/)
+- [`MongoDBDataProvider`](https://www.npmjs.com/package/@graphback/runtime-mongo): Data provider for MongoDB
+- [`KnexDBDataProvider`](https://www.npmjs.com/package/@graphback/runtime-knex): Data provider using [Knex.js](http://knexjs.org/)
 
-Your resolvers can use different data providers thanks to swapping implementation in resolver context
+Your resolvers can use any of these data providers by swapping the implementation and attaching it to your application context. See [Data Sources](../db/datasources) to learn how.
 
 ![](/img/runtime.png)
 
-## Adding custom resolvers to runtime
-
-Runtime layer will come with autogenerated schema that will also include custom resolvers provided in model.
-Developers can append their custom resolvers to resolver array to extend runtime layer.
-
-## How Runtime relates to the resolvers
```

Comment:
Is that moved somewhere? It feels an important piece of information

Reply:
Moved back